changebot/blueprints/stale_pull_requests.py

```diff
@@ -67,7 +71,9 @@ def process_pull_requests(repository, installation):
 
         if current_app.stale_pull_requests_close and dt > current_app.stale_pull_requests_close_seconds:
             comment_ids = pr.find_comments('astropy-bot[bot]', filter_keep=is_close_epilogue)
-            if len(comment_ids) == 0:
+            if enable_autoclose != 1:
```

Comment:
@bsipocz , is this what you had in mind in #46?

Reply:
Yes. Though a bool value for it may be more intuitive?